Resources/Prototypes/_DeadSpace/Entities/Objects/Specific/Chapel/censer.yml

```
@@ -35,5 +35,11 @@
     quickEquip: false
     slots:
     - Belt
-  - type: DrainableSolution
-    solution: drink
+  - type: Censer
+    heailng:
+      groups:
+        Brute: -45
+        Burn: -45
+        Airloss: -30
```

```suggestion
        Brute: -15
        Burn: -15
        Airloss: -15
```

Так будет более честно с учётом того что у тебя бесконечный запас святой воды + нет шанса на фейл хила